src/lib/markbind/src/patches/htmlparser2.js

```diff
@@ -261,6 +261,22 @@ Tokenizer.prototype._stateBeforeSpecial = function(c) {
 	this._special = result + 1;
 };
 
+/**
+ * Removes the special state if the special tag was self-closed.
```

Comment:
Can we explicitly state that this is an altered version of the `_stateInSelfClosingTag` defined in `htmlparser`? I think it might help other developers new to this part of the codebase understand what's going on better.